### PR TITLE
fix: getWidgetOptionSchema does not return undefined and separate more options related logics as composable

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Please review in order by commits! Commit message says it all.

### CODE REVIEW

File: apps/web/src/services/dashboards/dashboard-create/modules/dashboard-templates/helper.ts [hunk 1]
```diff
@@ -12,7 +12,12 @@ import { managedDashboardVariablesSchema, managedVariablesPropertiesMap } from '
 import type { DashboardLayoutWidgetInfo } from '@/services/dashboards/widgets/_configs/config';
 import { getWidgetConfig } from '@/services/dashboards/widgets/_helpers/widget-helper';
 
-
+export const getVariableKeyFromWidgetSchemaProperty = (property: string): string => {
+    if (property.startsWith('filters.')) {
+        return property.replace('filters.', '');
+    }
+    return property;
```
Comment (yuda110) [inline apps/web/src/services/dashboards/dashboard-create/modules/dashboard-templates/helper.ts:19]: question: Isn't it same with this?

```
return return property.replace('filters.', '');
```

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 2]
```diff
@@ -163,18 +123,25 @@ const handleReturnToInitialSettings = () => {
 };
 
 /* inherit */
-const handleChangeInheritToggle = (propertyName: string, value) => {
+const handleChangeInheritToggle = (propertyName: string, isInherit: boolean) => {
```
Comment (skdud4659) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue:126]: 👍🏻

**Review by skdud4659 — APPROVED** [review]
LGTM!

**Pull request merged**